
From: Nikita Danilov <nikita@clusterfs.com>

reiser4 wasn't updated during vfs_permission/generic_permission
conversion. Evil conspiracy is obviously underway.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/fs/reiser4/plugin/security/perm.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN fs/reiser4/plugin/security/perm.c~reiser4-generic_acl-fix fs/reiser4/plugin/security/perm.c
--- 25/fs/reiser4/plugin/security/perm.c~reiser4-generic_acl-fix	Tue Sep 14 15:51:43 2004
+++ 25-akpm/fs/reiser4/plugin/security/perm.c	Tue Sep 14 15:51:43 2004
@@ -13,7 +13,7 @@
 static int
 mask_ok_common(struct inode *inode, int mask)
 {
-	return vfs_permission(inode, mask);
+	return generic_permission(inode, mask, NULL);
 }
 
 static int
_
